### PR TITLE
upgrade crossterm 0.27 -> 0.28

### PR DESCRIPTION
**Pull request by boxdot**

### CODE REVIEW

File: examples/example-wifi.rs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#[allow(clippy::incompatible_msrv)]
```
Comment (timvisee) [inline examples/example-wifi.rs:1]: Rather than adding this, it's probably better to bump the MSRV defined in `Cargo.toml`. Wdyt?

**Review by timvisee — CHANGES_REQUESTED** [review]

File: examples/example-wifi.rs (hunk 1 above)
Comment (boxdot) [reply examples/example-wifi.rs]: sure.

**Comment by timvisee** [comment]
Thanks!

Would you prefer a new release on crates.io right away to include this?

**Review by timvisee — APPROVED** [review]

**Comment by timvisee** [comment]
I've just published version 0.3.2

**Pull request merged**